cmd/spiffe-helper/config/config.go

```diff
@@ -46,28 +48,36 @@ type JWTConfig struct {
 	JWTSVIDFilename string `hcl:"jwt_svid_file_name"`
 }
 
+func ParseFlags() (string, bool) {
+	configFile := flag.String("config", "helper.conf", "<configFile> Configuration file path")
+	daemonModeFlag := flag.Bool(daemonModeFlagName, true, "Exit once the requested objects are retrieved")
+	flag.Parse()
```

Comment:
For some reason, in my testing adding
```go
logrus.Infof("Parsed daemon flag of %t", *daemonModeFlag)
```
after this line logs true, even if I passed `false`, `FALSE`, or `0` on the CLI.